### PR TITLE
Sync up from adobe/express-website to adobecom/express

### PR DESCRIPTION
**Pull request by apganapa**

### CODE REVIEW

**Review by JingleH — DISMISSED** [review]

**Comment by JingleH** [comment]
I really wanna get the main synched, so then we can get the stage synched, and then we can port template-list and templates.js performance improvements again back from adobe/express-website to adobecom/express 😢 this is kindda blocking my and @qiyundai 's work at the moment

File: tools/sidekick/config.json [hunk 1]
```diff
@@ -1,20 +1,22 @@
 {
   "project": "Express",
+  "outerHost": "express-website--adobe.hlx.live",
```
Comment (vhargrave) [inline tools/sidekick/config.json:3]: I don't think these changes are wanted either - maybe just the feds-header

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -3,5 +3,5 @@ Please always provide the [JIRA issue(s)](https://jira.corp.adobe.com/secure/Rap
 Fix <jira-issue-id>
 
 Test URLs:
```
Comment (vhargrave) [inline .github/pull_request_template.md:5]: we don't want these changes either

**Review by vhargrave — CHANGES_REQUESTED** [review]
I'm almost wondering if it would make sense to create a branch off `adobecom:main`, where you test the merge from `adobe:main`. Once everything's working and tested, then you merge from that branch to `adobe:main`. 

@JingleH @qiyundai @apganapa thoughts?

**Comment by apganapa** [comment]
https://github.com/adobecom/express/tree/MWPW-132431-from-main is created from https://github.com/adobecom/express
Post merging, individual files fstab.yaml file and config.json will be modified.

**Pull request merged**